workos/mfa.py

```diff
@@ -127,6 +126,11 @@ def verify_factor(
         Returns: Dict containing the  challenge factor details.
```

Comment:
Should we also update the Docstring for `verify_factor` to mention that is now deprecated?

Reply:
Done!